src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -602,19 +602,19 @@ export default class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang
 
 <style lang="scss">
 .tree-builder-toolbar {
-  position: sticky;
```

Comment:
But I thought `sticky` was our newest bff!! 👯

Reply:
This was removed as it was redundant code that came in from Saf's changes. I have used this style on `interaction-designer` rather than using it in the Toolbar to wrap it around the whole Toolbar rather than pieces of it.